### DISCUSSION TITLE
Using "Headreco"

### DISCUSSION THREAD

**Discussion by tmedani** [simnibs/simnibs#340] [2. General]
Dear SimNibs team,

I'm wondering if it is still possible to use the "Headreco" process.
And how to get access to it. 

We are still using this process for other applications. 

Thank you,

Best,
Takfarinas

**Comment by oulap** [comment]
Hi Takfarinas,

You can still find the old versions of simnibs with headreco under "releases", see here:

https://github.com/simnibs/simnibs/releases

However, headreco is not part of v4.0 and up.

**Reply by tmedani** [reply]
Awsome, thank you @oulap for the conformation
